internal/plugins/golang/v2/plugin.go

```diff
@@ -45,9 +45,11 @@ var (
 // add operator-framework features to the base kubebuilder Go scaffold and CLI.
 type Plugin struct{}
 
-func (Plugin) Name() string                       { return (kbgov2.Plugin{}).Name() }
-func (Plugin) Version() plugin.Version            { return (kbgov2.Plugin{}).Version() }
-func (Plugin) SupportedProjectVersions() []string { return (kbgov2.Plugin{}).SupportedProjectVersions() }
+func (Plugin) Name() string            { return (kbgov2.Plugin{}).Name() }
+func (Plugin) Version() plugin.Version { return (kbgov2.Plugin{}).Version() }
+func (Plugin) SupportedProjectVersions() []string {
+	return (kbgov2.Plugin{}).SupportedProjectVersions()
```

Comment:
required for go 1.14+ changes ^ ( lint will not pass because it is doing the 14 checks)

Reply:
We haven't bumped to go 1.14 so we shouldn't change this yet.

Reply:
The latest version of the lint checks the go fmt with this fix. So, if we do not fix the fmt it will not pass. (ps it do not change the current behaviour and go fmt for 1.13 accepted that as well.)

Reply:
Yes the golangci-lint does 1.14 checks. Venkat hit this with his PR, so we used the older version.

Reply:
Just to clarifies: 

We are just changing the fmt of 3 lines according to go rules which are valid for 1.13 and 1.14. See that Travis is using 1.13 and passed in the git diff test of the sanity check. (btw, done automatically when we run make lint-fix too). 

Also, my motivation for this pr is not be required to delete/install the lint when we switch to work in other projects besides solving technical debts. The upper version works well if the project is using a lower one but not the opposite.